### PR TITLE
feat(website): add FAQ header styling

### PR DESCRIPTION
**Pull request by theosanderson**
Used for https://github.com/pathoplexus/pathoplexus/pull/63

### CODE REVIEW

File: website/src/styles/mdcontainer.scss [hunk 1]
```diff
@@ -123,3 +123,12 @@
 .mdContainer summary{
     @apply cursor-pointer;
 }
+
+.mdContainer  .faqWrapper h3{
```
Comment (chaoran-chen) [inline website/src/styles/mdcontainer.scss:127]:
```suggestion
.mdContainer .faqWrapper h3{
```

File: website/src/styles/mdcontainer.scss [hunk 2]
```diff
@@ -123,3 +123,12 @@
...
+    @apply text-lg font-semibold text-main my-3;
+    border-top: 1px solid #ddd;
+    margin-top: 1.75rem;
+    padding-top: 1.25rem;
+    margin-bottom:0;
```
Comment (chaoran-chen) [inline website/src/styles/mdcontainer.scss:132]:
```suggestion
    margin-bottom: 0;
```

Comment (chaoran-chen) [inline website/src/styles/mdcontainer.scss:132]: Would it be better to not mix the use of tailwind classes and raw CSS and stick with one type as much as possible?

**Review by chaoran-chen — APPROVED** [review]

**Comment by emmahodcroft** [comment]
I really like the dividers on the front page FAQ 👍

**Comment by theosanderson** [comment]
Thanks Chaoran! (and Emma!)

**Pull request merged**